kale/sqs.py

```diff
@@ -76,7 +77,7 @@ def _get_or_create_queue(self, queue_name):
                 raise e
 
             logger.info('Creating new SQS queue: %s' % queue_name)
-            queue = self._client.create_queue(QueueName=queue_name)
+            queue = self._client.create_queue(QueueName=queue_name, tags={"dlq":str(is_dlq)})
```

Comment:
Tags looks to be lower case in the Boto3 docs, however inconsistent. https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.create_queue